KarateTraining/src/test/java/karate-config.js

```diff
@@ -0,0 +1,26 @@
+function fn() {
+  var env = karate.env; // get system property 'karate.env'
+  var jsonConfig = read('classpath:src/main/resources/qa.json');
+  karate.log('karate.env system property was:', env);
+  if (!env) {
+    env = 'qa';
+    var config = {
+        env: env,
+        baseUrl: jsonConfig.baseQaUrl
+      }
+  }
+
+  if (env == 'dev') {
+  var config = {
+          env: env,
+          baseUrl: jsonConfig.baseDevUrl
+        }
+      } else if (env == 'stg') {
+      var config = {
+              env: env,
+              baseUrl: jsonConfig.baseStgUrl
+            }
+      }
+
+  return config;
```

Comment:
if you gonna use just a file to read the configs (`qa.json`), get the correct data inside the file from the envinroment. Something like this:
`var config = jsonConfig[env];`